src/logging.cpp

```diff
@@ -203,9 +203,9 @@ std::string BCLog::Logger::LogTimestampStr(const std::string& str)
             strStamped.pop_back();
             strStamped += strprintf(".%06dZ", nTimeMicros%1000000);
         }
-        int64_t mocktime = GetMockTime();
-        if (mocktime) {
-            strStamped += " (mocktime: " + FormatISO8601DateTime(mocktime) + ")";
+        std::chrono::seconds mocktime = GetMockTime();
+        if (mocktime.count() == 0) {
+            strStamped += " (mocktime: " + FormatISO8601DateTime(mocktime.count()) + ")";
```

Comment:
```suggestion
            strStamped += " (mocktime: " + FormatISO8601DateTime(count_seconds(mocktime)) + ")";
```

This will prevent silent surprises if `GetMockTime()` is changed some day to return something else than `std::chrono::seconds`, e.g. `std::chrono:milliseconds`.

Reply:
cool, didn't know about these helpers. updated. thanks!